### ISSUE TITLE
rowHeight larger than 70dp does not work well

### ISSUE THREAD

**Issue by gilgil28** [edsilfer/sticky-index#3]
First of all, great library!

There seem to be a problem with rowHeight larger than 70dp. The indices seem to be misplaced and laggy, or "jumpy"

**Comment by gilgil28** [comment]
also, why is it demanding minSDK 22? I downloaded the project and it works fine with lower versions (checked 15)

**Comment by edsilfer** [comment]
Hi @gilgil28, I'm glad that this library is somehow usefull to you. The problem with the height, I believe, is being caused due the way I am recoverying the dp value and setting it to the height (I might need to use ´´´TypedValue.COMPLEX_UNIT_PX´´´ when setting the new layout height). The min sdk version is due my creation process in AS, I likely forgot to change it before uploading the file. I took note of our comments and once I have sometime (perhaps today yet) I'll try to fix it and make a new release. Thank you for your comments

**Comment by edsilfer** [comment]
Hi @gilgil28 , I believe your bug is fixed in the verion 1.2.0. Can you please try it out and tell me the result?

**Comment by gilgil28** [comment]
thanks for the update! can you make the minimum SDK version lower tha 22? maybe 15?

**Comment by gilgil28** [comment]
I tried it and the sticky letters are all gone, appear only sometimes. probably a problem with the alpha parameter

**Comment by edsilfer** [comment]
Hello @gilgil28 , I am sorry for the trouble but all my tests are passing now. You were right about the alpha, however it wasn't easy to spot a solution for it - I hope now it works fine. I released a new version with this bug fix 1.2.1. Just tell me if it works now. Also, not that I've added the new attribute ```app:stickyWidth``` that allows you to control the distante between the text and the parent corner (details in the read-me file).
